packet_handlers.go

```diff
@@ -386,6 +386,33 @@ func (p *Parser) handleGameEvent(ge *msg.CSVCMsg_GameEvent) {
 			HasKit:  pl.HasDefuseKit,
 		})
 
+	case "item_equip": // Equipped, I think
+		fallthrough
+	case "item_pickup": // Picked up or bought?
+		fallthrough
+	case "item_remove": // Dropped?
+		data = mapGameEventData(d, ge)
+		player := p.connectedPlayers[int(data["userid"].GetValShort())]
+		weapon := common.NewSkinEquipment(data["item"].GetValString(), "")
+
```

Comment:
Nested structs as well, see buildNadeEvent() - but don't do be like me, that function really belongs into the events package. (I'll move the buildNadeEvent one as well now that I saw that).